certutil/bundler.go

```diff
@@ -402,13 +402,13 @@ func constructCertFileName(cert *x509.Certificate) string {
 func (b *Bundler) fetchIntermediates(certs []*x509.Certificate) (err error) {
 	if IntermediateStash != "" {
 		if _, err := fileutil.Vfs.Stat(IntermediateStash); err != nil && os.IsNotExist(err) {
-			logger.Infof("intermediate stash directory %s doesn't exist, creating", IntermediateStash)
+			logger.KV(xlog.INFO, "intermediate stash directory doesn't exist, creating", IntermediateStash)
 			err = fileutil.Vfs.MkdirAll(IntermediateStash, 0755)
 			if err != nil {
-				logger.Errorf("failed to create intermediate stash directory %s: %v", IntermediateStash, err)
+				logger.KV(xlog.ERROR, "failed to create intermediate stash directory", IntermediateStash, ":", err)
```

Comment:
`logger.KV(xlog.ERROR, "reason", "failed to create intermediate stash directory", "fodler", IntermediateStash, "err", err)